pkg/repository/prisma/dbsqlc/workflow_runs.sql

```diff
@@ -519,6 +519,43 @@ INSERT INTO "WorkflowRun" (
     sqlc.narg('priority')::int
 ) RETURNING *;
 
+
+-- name: CreateWorkflowRuns :copyfrom
+INSERT INTO "WorkflowRun" (
+    "id",
+    "displayName",
+    "tenantId",
+    "workflowVersionId",
+    "status",
+    "childIndex",
+    "childKey",
+    "parentId",
+    "parentStepRunId",
+    "additionalMetadata",
+    "priority"
+) VALUES (
+    $1,
+    $2,
+    $3,
+    $4,
+    $5,
+    $6,
+    $7,
+    $8,
+    $9,
+    $10,
+    $11
+
+);
+
+
+-- name: GetInsertedWorkflowRuns :many
+
+SELECT * FROM "WorkflowRun"
+WHERE xmin::text = (txid_current() % (2^32)::bigint)::text
```

Comment:
Let's not use the `xmin` hack unless we have a good way of avoiding seq scans on the whole table?

Reply:
sorry was rushing to get in in on Friday and overlooked this we happy with the following . We have an index on createdAt so we should just be using the xmin to filter a small result set. 

-- name: GetInsertedWorkflowRuns :many

SELECT * FROM "WorkflowRun"
WHERE xmin::text = (txid_current() % (2^32)::bigint)::text
AND "createdAt" >= @createdAt::timestamp
ORDER BY id;